pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -293,6 +293,14 @@
         "4.0.1.0": "4.6.0"
       }
     },
+    "System.Windows.Extensions": {
+      "InboxOn": {
+        "netcoreapp3.0": "4.6.4.0"
+      },
+      "AssemblyVersionInPackageVersion": {
+        "4.0.1.0": "4.6.0"
```

Comment:
Either this says what the last package was, in which case this version doesn't exist; or it says the current version, in which case it doesn't match the props file for the package.

Either way, I feel like there's something not right about this line.

Reply:
We should start out with assembly version of 4.0.0.0, please set appropriately in dir.props